spec/manager_spec.rb

```diff
@@ -1,9 +1,152 @@
 require "webmock/fixtures"
 
 describe WebMock::Fixtures::Manager do
-  describe "constructor" do
-    it "should not fail" do
-      WebMock::Fixtures::Manager.new()
+  after(:example) do
+    # Ensure we remove any added fixtures between tests
+    WebMock::Fixtures::Manager.reset!
+  end
+
+  before(:example) do
+    # Register a single fixture before each test
+    WebMock::Fixtures::Manager.register_fixture(:get_example, :get, %r{www.example.org}, :body => "Hello World")
+  end
+
+  describe "::reset!" do
+    it "should remove any registered fixtures" do
+      expect(WebMock::Fixtures::Manager.registered_fixtures[:get_example]).to_not(be_nil)
+      WebMock::Fixtures::Manager.reset!
+      expect(WebMock::Fixtures::Manager.registered_fixtures).to(eq({}))
+    end
+  end
+
+  describe "::register_fixture" do
+    context "with a subclassed manager" do
+      it "should not share registered fixtures" do
+        class TestManager < WebMock::Fixtures::Manager
+        end
+        TestManager.register_fixture(:get_google, :get, %r{www.google.com}, :body => "Google")
+
+        # Ensure TestManager has the appropriate fixtures
+        expect(TestManager.registered_fixtures.keys()).to(eq([:get_google]))
+        expect(TestManager.registered_fixtures[:get_google]).to_not(be_nil)
+
+        # Ensure WebMock::Fixtures::Manager has the appropriate fixtures
+        expect(WebMock::Fixtures::Manager.registered_fixtures.keys()).to(eq([:get_example]))
+        expect(WebMock::Fixtures::Manager.registered_fixtures[:get_google]).to(be_nil)
+      end
+    end
+
+    it "should properly register the fixture" do
+      expected = {
+        :pattern => %r{www.example.org},
+        :verb => :get,
+        :response => {
+          :body => "Hello World",
+        },
+      }
+      expect(WebMock::Fixtures::Manager.registered_fixtures[:get_example]).to(eq(expected))
+    end
+  end
+
+  describe "::load_fixture_file" do
+    before(:example) do
+      fixture_path = File.join(File.expand_path(File.dirname(__FILE__)), "get_google_200.raw")
+      WebMock::Fixtures::Manager.load_fixture_file(:get_google, :get, %r{www.google.com}, fixture_path)
+    end
+
+    it "should properly register a fixture" do
+      fixture = WebMock::Fixtures::Manager.registered_fixtures[:get_google]
+      expect(fixture[:verb]).to(eq(:get))
+      expect(fixture[:pattern]).to(eq(%r{www.google.com}))
+
+      # Assert that we read the contents of the file
+      expect(fixture[:response]).to(be_a(String))
+      expect(fixture[:response]).to(start_with("HTTP/1.1 200 OK"))
+    end
+  end
+
+  describe "::run" do
+    context "with an unknown fixture name" do
+      it "should raise an error" do
+        expect { WebMock::Fixtures::Manager.run([:unknown_fixture]) }.to(raise_error(KeyError))
```

Comment:
The spaces around this block is inconsistent. We should use either `expect { } .to`, `expect{ }.to`, or `expect({ }).to`